### DISCUSSION TITLE
Table driven scenarios are skipped when executing the gauge run specs/ cmd

### DISCUSSION THREAD

**Discussion by ravikiranhebbar88** [getgauge/gauge#2171] [General]
Table-driven scenarios are skipped when executed with the gauge run specs/ cmd.

![image](https://user-images.githubusercontent.com/12529893/143682328-94d92dd6-33c9-4c35-95af-b10f92939e30.png)

Table-driven scenarios are working when it is manually executed by clicking on **Run Scenario** link in Visual basic code.

![image](https://user-images.githubusercontent.com/12529893/143682432-5a8e3af3-0a23-4ebc-ae9b-0356e7600cc9.png)

![image](https://user-images.githubusercontent.com/12529893/143682449-f4cacad0-1fcd-49ab-b05c-02c1505afd36.png)

And, it is not displaying anything in the logs why the scenario is skipped.

Please let me know.

**Comment by sriv** [comment]
please check if your project has `allow_scenario_datatable ` set to true

**Reply by ravikiranhebbar88** [reply]
> please check if your project has `allow_scenario_datatable ` set to true

it is set to true

![image](https://user-images.githubusercontent.com/12529893/143682771-d08046a9-5683-4009-a615-0ad48341e815.png)

**Reply by sriv** [reply]
are you running `gauge run specs` with any additional flags?

**Reply by ravikiranhebbar88** [reply]
not using any flags

![image](https://user-images.githubusercontent.com/12529893/143683254-221d68e1-9add-4cbc-99be-ff55229b5911.png)

**Reply by sriv** [reply]
[baz.zip](https://github.com/getgauge/gauge/files/7611832/baz.zip)

I am on 

```
> gauge -v
Gauge version: 1.4.1
Commit Hash: a82478a

Plugins
-------
dotnet (0.4.1)
html-report (4.1.2)
java (0.7.15)
js (2.3.17)
json-report (0.3.6)
screenshot (0.1.0)
xml-report (0.2.4)
```

See sample project attached with table driven scenario. I can see it getting executed when I use `gauge run specs` as well as `npm test` 

It's strange that you are seeing this behaviour.

**Reply by ravikiranhebbar88** [reply]
okay, Thanks!!!. Will check

**Reply by ravikiranhebbar88** [reply]
@sriv Table-driven scenarios were not skipped. The scenarios were failing as it was not executing the scenarios in order. It is working fine after adding the table for each scenario within a spec.
